pkg/cmd/cmd.go

```diff
@@ -55,62 +59,20 @@ func (cmd *Command) Execute(ctx context.Context) error {
 		Scheme:             scheme,
 		MetricsBindAddress: "0",
 	})
-
 	if err != nil {
-		return fmt.Errorf("manager new: %w", err)
-	}
-
-	if err := registrar.RegisterControllers(mgr); err != nil {
-		return fmt.Errorf("register controllers: %w", err)
+		return fmt.Errorf("failed to create new manager: %w", err)
 	}
 
-	if err := registrar.IndexResources(ctx, mgr); err != nil {
```

Comment:
Does kubebuilder help us do this as well, or is this simply not needed anymore?

Reply:
Not needed anymore. This was leftover from back when we were finding matching supply chains for workloads differently, but the way we do it now there's no longer a need for this.